src/user_maintainance.c

```diff
@@ -34,4 +64,73 @@ char* login(char *u_id, char *pwd){
 			return "invalid";
 		}
 	}
-}
\ No newline at end of file
+}
+
+
+int show_exam_result(){
+
+	char s;
+	
+	FILE * fPtr;
+   	
+    fPtr = fopen("STUDENT_RESULTS.txt","r"); 
+    
+    if(fPtr == NULL)
+    {
+    	printf("Unable to read results. \n");
+    	exit(EXIT_FAILURE);
+    } 	
+	
+	while((s=fgetc(fPtr))!= EOF) {
+      printf("%c",s);
```

Comment:
This file only display a single character but overall detail, Like which topic name I think it should be 
Your Topic is= 
Attempt questions: 
Correct answers
Wrong Answers and total result